From 0b0ff040b1ae47878c08b86b56f459bf51aec468 Mon Sep 17 00:00:00 2001
From: Julien Jerphanion <git@jjerphan.xyz>
Date: Fri, 28 Apr 2023 17:40:07 +0200
Subject: [PATCH 3/3] build: Correct case for CMake config

Signed-off-by: Julien Jerphanion <git@jjerphan.xyz>
---
 RemoteryConfig.cmake.in | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/RemoteryConfig.cmake.in b/RemoteryConfig.cmake.in
index 128a8cf..2c04478 100644
--- a/RemoteryConfig.cmake.in
+++ b/RemoteryConfig.cmake.in
@@ -13,32 +13,32 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR};${CMAKE_MODULE_PATH}")
 
 @REMOTERY_CONFIG_CODE@
 
-if(NOT TARGET remotery AND NOT TARGET remotery-static)
+if(NOT TARGET Remotery AND NOT TARGET Remotery-static)
     include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
 
-    if (TARGET remotery AND TARGET remotery-static)
+    if (TARGET Remotery AND TARGET Remotery-static)
         get_target_property(
-            @PROJECT_NAME@_INCLUDE_DIR remotery INTERFACE_INCLUDE_DIRECTORIES
+            @PROJECT_NAME@_INCLUDE_DIR Remotery INTERFACE_INCLUDE_DIRECTORIES
         )
         get_target_property(
-            @PROJECT_NAME@_LIBRARY remotery LOCATION
+            @PROJECT_NAME@_LIBRARY Remotery LOCATION
         )
         get_target_property(
-            @PROJECT_NAME@_STATIC_LIBRARY remotery-static LOCATION
+            @PROJECT_NAME@_STATIC_LIBRARY Remotery-static LOCATION
         )
     elseif (TARGET remotery)
         get_target_property(
-            @PROJECT_NAME@_INCLUDE_DIR remotery INTERFACE_INCLUDE_DIRECTORIES
+            @PROJECT_NAME@_INCLUDE_DIR Remotery INTERFACE_INCLUDE_DIRECTORIES
         )
         get_target_property(
-            @PROJECT_NAME@_LIBRARY remotery LOCATION
+            @PROJECT_NAME@_LIBRARY Remotery LOCATION
         )
     elseif (TARGET remotery-static)
         get_target_property(
-            @PROJECT_NAME@_INCLUDE_DIR remotery-static INTERFACE_INCLUDE_DIRECTORIES
+            @PROJECT_NAME@_INCLUDE_DIR Remotery-static INTERFACE_INCLUDE_DIRECTORIES
         )
         get_target_property(
-            @PROJECT_NAME@_STATIC_LIBRARY remotery-static LOCATION
+            @PROJECT_NAME@_STATIC_LIBRARY Remotery-static LOCATION
         )
         set(
             @PROJECT_NAME@_LIBRARY ${@PROJECT_NAME@_STATIC_LIBRARY}
-- 
2.40.0

